src/app/core/theme.component.ts

```diff
@@ -1,8 +1,21 @@
 import { Component } from '@angular/core';
+import { FormResultViewComponent, FormResultInfo } from '@shared/ui';
 
 @Component({
   selector: 'app-theme',
+  imports: [FormResultViewComponent],
   standalone: true,
-  template: ` <h1>Storybook-like route</h1> `,
+  template: `
+    <h1>Storybook-like route</h1>
+    <app-form-result-view [formResultInfo]="formResultInfo">
+      <img src="../../../assets/form-result-icons/confirm-icon.svg" alt="" />
+      <button>-></button>
+    </app-form-result-view>
```

Comment:
Tu możnaby jeszcze dodać jakiś nagłówek czym jest ten komponent jest bo będziemy mieli dużo elementów w theme to będzie się łatwiej odnaleźć jeśli będą jakoś oddzielone albo zatytułowane